src/Header.js

```diff
@@ -0,0 +1,21 @@
+import { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+import './Header.css'
+
+class Header extends Component {
+    render() { 
+        return ( 
+            <header>
+                <h1>Pokedex</h1>
+                <div className='links'>
+                    <NavLink exact to='home'>
```

Comment:
Just missing the active styling on these NavLinks! You want the link to look different when it is the actively selected page so that the user has a visual of which page they're on.